src/test/locale/vi.js

```diff
@@ -27,14 +27,14 @@ test('parse', function (assert) {
 
     for (i = 0; i < 12; i++) {
         tests[i] = tests[i].split(',');
-        equalTest(tests[i][0], 'MMM', i);
-        equalTest(tests[i][1], 'MMM', i);
-        equalTest(tests[i][0], 'MMMM', i);
-        equalTest(tests[i][1], 'MMMM', i);
-        equalTest(tests[i][0].toLocaleLowerCase(), 'MMMM', i);
-        equalTest(tests[i][1].toLocaleLowerCase(), 'MMMM', i);
-        equalTest(tests[i][0].toLocaleUpperCase(), 'MMMM', i);
-        equalTest(tests[i][1].toLocaleUpperCase(), 'MMMM', i);
+        equalTest(tests[i][0], '[tháng] M', i);
```

Comment:
Why did these change? The tests should pass without the test changes in `equalTest()`.